api/catalog/api/serializers/media_serializers.py

```diff
@@ -48,10 +59,11 @@ class MediaSearchRequestSerializer(serializers.Serializer):
         "extension",
         "mature",
         "qa",
-        # "unstable__sort_by",  # excluding unstable fields
-        # "unstable__sort_dir",  # excluding unstable fields
-        # "unstable__authority",  # excluding unstable fields
-        # "unstable__authority_boost",  # excluding unstable fields
+        # Excluded unstable fields, also see `exclude_fields` above.
```

Comment:
Do we have to duplicate the list of excluded fields? What would happen if you removed the commented out fields here?

Reply:
This list is only used to generate the names of fields in the docs. They are listed here and commented so that they can be un-commented when the fields are finally considered stable.